backend/process_miner/logs_importer.py

```diff
@@ -0,0 +1,29 @@
+"""
+Module used for importing tagged logs into heuristic miner
+"""
+import glob
+import pandas as pd
+from pm4py.objects.log.adapters.pandas import csv_import_adapter
+
+from pm4py.objects.conversion.log import factory as conversion_factory
+
+from pm4py.algo.discovery.heuristics import algorithm as heuristics_miner
+from pm4py.visualization.heuristics_net import visualizer as hn_vis
+
+
+def concat_files():
+    all_filenames = [i for i in glob.glob('retrieved_logs/'+'*.csv')]
+    # combine all files in the list
+    combined_csv = pd.concat([pd.read_csv(f) for f in all_filenames])
+    # export to csv
+    combined_csv.to_csv("concated_files.csv", index=False, encoding='utf-8-sig')
```

Comment:
Instead of writing the dataframe to disk you could just directly use it in `execute_script`